app/Models/V2/AuditAttachment/AuditAttachment.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Models\V2\AuditAttachment;
+
+use App\Models\V2\AuditStatus\AuditStatus;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class AuditAttachment extends Model
+{
+    use HasFactory;
+    use SoftDeletes;
+
+    public $table = 'attachment';
+
+    protected $fillable = [
+        'entity_id',
+        'file_name',
+        'file_url',
+        'date_created',
+        'created_by',
+    ];
+
+    public function auditStatus()
+    {
+        return $this->belongsTo(AuditStatus::class, 'entity_id', 'id');
+    }
```

Comment:
As I've read more of the PR, I think this table is unneeded. We already use Spatie for attaching media of any type to a model, so this should use that system instead. Please see the `HasV2MediaCollection` and its uses as an example. `Organisation` in particular has lots of "media" that are files like spreadsheets, so it's not just images.